### PR TITLE
Push out grammar and index build reference/how-to

### PR DESCRIPTION
**Pull request by alch-msft**

### CODE REVIEW

File: academic-services/graph/define-mag-function.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+services: microsoft-academic-services
+ms.topic: tutorial
+ms.service: microsoft-academic-services
+ms.date: 9/23/2020
+---
+# Define MAG functions (U-SQL)
+
+## Define functions to extract MAG data
+
+In this section, you submit an Azure Data Lake Analytics (ADLA) job to create functions extracting MAG data from Azure Storage (AS).
```
Comment (RickRogahn) [inline academic-services/graph/define-mag-function.md:13]: >you submit [](start = 17, length = 10)

you will submit

File: academic-services/graph/define-mag-function.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+1. Download `samples/usql/CreateFunctions.usql` in MAG dataset to your local drive. <br> From [Azure portal](https://portal.azure.com), go to the Azure Storage account > **Containers > [mag-yyyy-mm-dd] > samples > usql > CreateFunctions.usql > Download**.
```
Comment (RickRogahn) [inline academic-services/graph/define-mag-function.md:15]: ><br> [](start = 84, length = 4)

forcing a carriage return in markdown?

**Comment by RickRogahn** [comment]
What are the major changes here that need to be reviewed?  Looks like we are pulling in changes from another branch that we have reviewed already...

**Comment by DarrinEide** [comment]
You need to update the "Customize MAKES" section in /academic-services/knowledge-exploration-service/index.yml with links to the new custom grammar stuff

**Review by DarrinEide — CHANGES_REQUESTED** [review]
:clock1:

**Review by DarrinEide — APPROVED** [review]
:shipit:

**Pull request merged**